ui/pages/permissions-connect/flask/snap-install/snap-install.js

```diff
@@ -37,15 +38,19 @@ export default function SnapInstall({
     approveSnapInstall,
   ]);
 
-  const shouldShowWarning = useMemo(
-    () =>
-      Boolean(
-        request.permissions &&
-          Object.keys(request.permissions).find((v) =>
-            v.startsWith('snap_getBip44Entropy_'),
-          ),
-      ),
-    [request.permissions],
+  const findBip44EntropyPermission =
+    request.permissions &&
+    Object.keys(request.permissions).find((v) =>
+      v.startsWith('snap_getBip44Entropy_'),
+    );
+
+  const shouldShowWarning = useMemo(() => Boolean(findBip44EntropyPermission), [
```

Comment:
Does this need `useMemo`? These operations shouldn't be that expensive, but I am not aware of the standards for the MM extension in general.

Reply:
It was already like this before, I don't know why it's using a memorised value.

Reply:
Okay, I don't see a reason for it to be memoized though.